src/compass/utils/helpers.py

```diff
@@ -348,10 +348,17 @@ def open_raster(filename, band=1):
         raise FileNotFoundError(err_str)
 
     try:
+        # The pythonic execption handling for GDAL can be turned on / off
+        # The flag of which can be identified by `gdal.GetUseExceptions()`
+        # In pythonic exception halding, `gdal.Open()` will raise `RuntimeError`
+        # In traditional GDAL, the function does not raise exception buy will return `None`,
+        # and the attempts to call GDAL methods will raise `AttributeError`
+
         ds = gdal.Open(filename, gdal.GA_ReadOnly)
         arr = ds.GetRasterBand(band).ReadAsArray()
         return arr
-    except AttributeError:
+
+    except (AttributeError, RuntimeError):
```

Comment:
Since GDAL has surprised us once, shall we get to except regardless of the error type?
```suggestion
    except :
```